src/libtrx/game/collision.c

```diff
@@ -7,6 +7,11 @@
 #include "game/rooms.h"
 #include "utils.h"
 
+#define LARA_HEADROOM 160 // Additional collision space above head.
```

Comment:
Module level constants are named with a M_ prefix to mark them as local to this file.